cli.js

```diff
@@ -4,29 +4,95 @@
 /* eslint no-console:0 */
 
 const katex = require("./");
-let input = "";
+const fs = require('fs');
 
 // Skip the first two args, which are just "node" and "cli.js"
 const args = process.argv.slice(2);
 
 if (args.indexOf("--help") !== -1) {
     console.log(process.argv[0] + " " + process.argv[1] +
                 " [ --help ]" +
-                " [ --display-mode ]");
+                " [ --display-mode ]" +
+                " [ --throw-on-error ]" +
+                " [ --error-color color ]" +
+                " [ --max-size size ]" +
+                " [ --input inputFile ]" +
+                " [ --output outputFile ]"
+               );
 
     console.log("\n" +
                 "Options:");
-    console.log("  --help            Display this help message");
-    console.log("  --display-mode    Render in display mode (not inline mode)");
+    console.log("  --help              Display this help message");
+    console.log("  --display-mode      Render in display mode (not inline mode)");
+    console.log("  --no-throw-on-error Render unsupported commands as text");
+    console.log("  --error-color       Color unsupported commands are rendered in");
```

Comment:
The help message should describe what format the color must be in and give an example.

Reply:
Initially I had all that, but then the line length was larger than 84 characters, so I dropped it.
If we readd the information, then the description must span multiple lines.

Reply:
Done, but not sure if this is better.

Reply:
Whenever we upgrade the option parser it would be nice if we could find one which supported short and long help strings and nicely formatted the longs ones like descriptions underneath the short one and properly aligned them.  Maybe commander.js already does, but it doesn't show in the given examples.